examples/bayesopt_example.py

```diff
@@ -0,0 +1,313 @@
+import warnings
+warnings.filterwarnings('ignore')
+
+import numpy as np
+import functorch as ft
+import torch
+from torch import nn, optim
+from torch import distributions as dists
+from torch.nn import functional as F
+from gpytorch import distributions as gdists
```

Comment:
Since there are multiple packages used here which we don't require for the laplace package itself, we might want to add an additional `requirements.txt` file or similar for the example scripts.

Reply:
[7726b3d](https://github.com/AlexImmer/Laplace/pull/120/commits/7726b3d9ee431e2a6c258ab27ddecd3b60d17b56)